client/src/Bit.js

```diff
@@ -24,12 +24,12 @@ export default function Bit(props) {
       <div>
         <BitTagPills bit={bit} {...props} />
         <p>
-          <b>{bit.get('author').get('name')}</b>
+          <b>{bit.get('author', new Map()).get('name')}</b>

```

Comment:
i don't understand the new Map() bit... but i'll just assume you know what you're doing.